### PR TITLE
github challenge

### PR DESCRIPTION
**Pull request by webcubed**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/_github-pages-challenge-webcubed.json
Comment (DEV-DIBSTER) [inline domains/_github-pages-challenge-webcubed.json:1]: Missing a subdomain in this.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/_github-pages-challenge-webcubed.json
Comment (webcubed) [reply domains/_github-pages-challenge-webcubed.json:1]: oh my bad gimie sec

Comment (webcubed) [reply domains/_github-pages-challenge-webcubed.json:1]: wait i'm confused what

Comment (webcubed) [reply domains/_github-pages-challenge-webcubed.json:1]: > Missing a subdomain in this.

what am i missing?

Comment (DEV-DIBSTER) [reply domains/_github-pages-challenge-webcubed.json:1]: Oh wait I think I figured it out, you should be good.

**Review by DEV-DIBSTER — APPROVED** [review]

File: domains/_github-pages-challenge-webcubed.json
Comment (webcubed) [reply domains/_github-pages-challenge-webcubed.json:1]: lol i panicked

Comment (iostpa) [reply domains/_github-pages-challenge-webcubed.json:1]: Sorry to break the news, but you needed to add the subdomain at the end of the file. Like in between `_github-pages-challenge-webcubed` and `.json`.
I mostly blame github for this though. Anyhow, I'd recommend making a new PR that will fix this.

Comment (webcubed) [reply domains/_github-pages-challenge-webcubed.json:1]: ah ok

Comment (iostpa) [reply domains/_github-pages-challenge-webcubed.json:1]: By blame I mean for not including the subdomain to where you copy and paste it

Comment (webcubed) [reply domains/_github-pages-challenge-webcubed.json:1]: i opened a new pr

Comment (webcubed) [reply domains/_github-pages-challenge-webcubed.json:1]: ty!

**Pull request merged**